pom.xml

```diff
@@ -90,6 +90,10 @@
                 <enabled>false</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>microprofile-config-staging</id>
+            <url>https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1375/</url>
+        </repository>
```

Comment:
This is temporary to be able to use MP Config 2.0 Staging binaries. Will be removed once all final versions are out.